internal/backend/local/test.go

```diff
@@ -781,6 +781,9 @@ func (runner *TestFileRunner) apply(tfCtx *terraform.Context, plan *plans.Plan,
 		defer done()
 		log.Printf("[DEBUG] TestFileRunner: starting apply for %s/%s", file.Name, run.Name)
 		updated, newScope, applyDiags = tfCtx.ApplyAndEval(plan, config, nil)
+		// if updated == nil {
```

Comment:
This might be related to the TODO lower down, but maybe warrants a TODO/FIXME to find it again later if you want to keep it in?